README.md

```diff
@@ -183,28 +183,39 @@ Mounting the Engine will provide the basic routes to authenticating a shop with
 |POST    |'/webhooks/:type'              |Webhook Callback              |
 
 
-The default routes of the Shopify rails engine, which is mounted to the root, can be altered to mount on a different route. The `config/routes.rb` can be modified to put these under a nested route (say `/app-name`) as:
+If required the engine can be mounted at a nested route, eg:
 
 ```ruby
 mount ShopifyApp::Engine, at: '/nested'
 ```
 
-This will create the Shopify engine routes under the specified subpath, as a result it will redirect new consumers to `/nested/login`. If you mount the engine at a subpath you'll also need to update the omniauth initializer to include a custom `callback_path` e.g:
+or
 
 ```ruby
-  provider :shopify,
-    ShopifyApp.configuration.api_key,
-    ShopifyApp.configuration.secret,
-    scope: ShopifyApp.configuration.scope,
-    callback_path: '/nested/auth/shopify/callback'
+namespace :nested do
```

Comment:
What do we think about changing that namespace to `shopify_app` ? I like that because it gives developers immediate context on what's being accessed.